src/Microsoft.AspNet.Testing/xunit/OperatingSystems.cs

```diff
@@ -13,7 +13,6 @@ public enum OperatingSystems
         Win2008R2 = 1 << 1,
         Linux = 1 << 2,
         MacOSX = 1 << 3,
-
-        Win7And2008R2 = Win7 | Win2008R2,
+        AllWindows = 1 << 4
```

Comment:
All Windows should also set `Win2008R2` and `Win7`. Now that I'm thinking maybe we should add Win8 and Win10 and just make `Windows` a mask for all of them